pkg/testgridanalysis/testgridanalysisapi/types.go

```diff
@@ -66,9 +66,10 @@ type OperatorState struct {
 }
 
 const (
-	OperatorUpgradePrefix       = "Operator upgrade "
-	OperatorFinalHealthPrefix   = "operator conditions "
-	FinalOperatorHealthTestName = `[sig-sippy] tests should finish with healthy operators`
+	OperatorUpgradePrefix        = "Operator upgrade "
+	OldOperatorFinalHealthPrefix = "operator conditions "
+	OperatorFinalHealthPrefix    = "Operator results.operator conditions "
```

Comment:
@bparees is there a better way to account this `Operator results.`?

Reply:
@deads2k added those so i'll defer to him.  you're looking for a way to identify those tests?  Or you're looking for the best way to assess final operator health?

Reply:
I'm looking for a way to distinguish infra failures (the installer wasn't able to do anything: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.8-upgrade-from-stable-4.7-e2e-ovirt-upgrade/1378693925845340160) and installation failures (some operators failed to proceed: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.8-upgrade-from-stable-4.7-e2e-ovirt-upgrade/1378693951728390144).

The signal isn't very reliable though, sometimes CI says that all operators are healthy when they are not: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.8-e2e-aws-serial/1379268056181641216

Reply:
I guess I can try to re-use `openshiftSyntheticManager` and all those abstraction layers